mushroomsoft/app/src/app/components/section/section.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="ms-section"> 
+    <div class="ms-container">
+        <div class="ms-motto-wrap">
+            <div class="ms-heading-jumbo-small">
```

Comment:
can you just have one div?

Reply:
no because the two div are different containers